src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBResource.cs

```diff
@@ -24,6 +25,23 @@ public class AzureCosmosDBResource(string name, Action<AzureResourceInfrastructu
     /// </summary>
     public BicepSecretOutputReference ConnectionString => new("connectionString", this);
```

Comment:
Is this property used anymore? I don't see it being used now that we have `internal BicepSecretOutputReference? ConnectionStringSecretOutput { get; set; }`.

Reply:
I don't think it is, but it's public so couldn't remove

Reply:
As a reminder, it started with you asking to change the type to `BicepOutputReference`, which we couldn't so we don't break, and we created the new one.

Reply:
Can we make it nullable and log it as a breaking change?

Reply:
A very minor breaking change that I don't think would affect anyone.